tt_eager/tt_dnn/op_library/backward/backward_ops.cpp

```diff
@@ -720,14 +720,14 @@ std::vector<Tensor> squared_difference_bw(const Tensor& grad, const Tensor& inpu
 
 // torch reference
 // - name: ldexp(Tensor self, Tensor other) -> Tensor
-//   self: 2^other
-//   other: self * ln(2) * (2^other)
-// # M_LN2 = ln(2)= 0.693147180s559945309417
+//   self: grad * 2^other
+//   other: grad * self * ln(2) * (2^other)
+// # M_LN2 = ln(2)= 0.693147180559945309417
 std::vector<Tensor> _ldexp_bw(const Tensor& grad, const Tensor& input, const Tensor& other, const MemoryConfig& output_mem_config) {
     std::vector<Tensor> grad_tensor;
-    Tensor tpow_o = mul_unary(exp(other, output_mem_config), M_LN2, output_mem_config);
+    Tensor tpow_o = mul(grad, rpow(other, 2.0, output_mem_config), std::nullopt, output_mem_config);
     grad_tensor.emplace_back(tpow_o);
-    Tensor result = mul(input, mul_unary(tpow_o, M_LN2, output_mem_config), std::nullopt, output_mem_config);
+    Tensor result = mul(grad, mul(input, mul_unary(tpow_o, M_LN2, output_mem_config), std::nullopt, output_mem_config), std::nullopt, output_mem_config);
```

Comment:
```suggestion
    Tensor result = mul(input, mul_unary(tpow_o, M_LN2, output_mem_config), std::nullopt, output_mem_config);
```
The original formula multiplies the gradient by two.  So I removed one multiplication of the gradient.
Because the gradient is already multiplied by `tpow_o`.

In my opinion, it seems that 'compare_results' is not correctly comparing the results.
Do you have any plans to fix this?

Reply:
@hschoi4448 In Compare results, there are two comparisons available, one is checking for PCC and another one will check for close all (few ops require close all check than PCC). If the op passes any one of the comparisons the function returns pass. 

Will keep this item, updating the compare_results in ToDo list.

Reply:
Updated the changes in the logic